src/theme/index.ts

```diff
@@ -104,6 +105,7 @@ const theme: any = {
     CustomSelect,
     DatePicker,
     FeedbackBox,
+    FeaturedContent,
```

Comment:
@EdwinGuzman I'm not seeing an example of something like `FeaturedContentProps` being imported into this file, did you mean what I have here?

Reply:
Ah, this is okay for the theme file import, but I meant for the component and type import/export in `src/index.ts`. Both files are named `index.ts` so it's confusing. The one in the `src` directory is to export the component for use in a consuming app. This one in `src/theme` is for Chakra's use.

Reply:
Ohh sorry should have read more carefully!